### PR TITLE
Add Opensuse dbus and pam whitelisting checks

### PR DESCRIPTION
**Pull request by acipm**
Adds dbus services whitelisting and ghost file checks in `DBusServicesCheck`.
Adds check for pam ghost files in `PAMModulesCheck`.

### CODE REVIEW

**Comment by mgerstner** [comment]
looks good to me

File: rpmlint/checks/DBusServicesCheck.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from rpmlint.checks.AbstractCheck import AbstractCheck
+
+
+class DBusServicesCheck(AbstractCheck):
+    _dbus_system_paths = [
```
Comment (marxin) [inline rpmlint/checks/DBusServicesCheck.py:5]: Please remove the leading `_`.

File: rpmlint/checks/DBusServicesCheck.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        self.services_whitelist = config.configuration['DBUSServices']['WhiteList']
+
+    def check(self, pkg):
+        if pkg.is_source:
+            return
+
+        for f in pkg.files:
+            for p in self._dbus_system_paths:
+                if f.startswith(p):
+
```
Comment (marxin) [inline rpmlint/checks/DBusServicesCheck.py:22]: Remove the extra new line.

File: rpmlint/descriptions/DBusServicesCheck.toml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+AUDIT_BUG_URL = "https://en.opensuse.org/openSUSE:Package_security_guidelines#audit_bugs"
```
Comment (marxin) [inline rpmlint/descriptions/DBusServicesCheck.toml:1]: Please create a new `rpmlint/descriptions/Variables.toml` file and mode there `AUDIT_BUG_URL` and remove all other definitions of it `git grep 'AUDIT_BUG_URL = '`.

**Review by marxin — CHANGES_REQUESTED** [review]

**Comment by marxin** [comment]
Thank you for the pull request.

**Pull request merged**